_episodes/19-motivation.md

```diff
@@ -243,7 +243,9 @@ to alienate a classroom and cause learners to tune out.
 
 ### Stereotype Threat
 
-Another demotivator is *[stereotype threat][wikipedia-stereotype-threat]*.
+Another demotivator is *[stereotype threat][wikipedia-stereotype-threat]*:
+a self-confirming belief that one may be evaluated based on a negative stereotype
```

Comment:
As quoted, yes. I am not a native speaker so I have to leave it up to others to make a final call here...